### ISSUE TITLE
Two way communication between front end and back end

### ISSUE THREAD

**Issue by LunaAsefaw** [kartoza/sawps#1936]
If users or managers are deleted here, they are not deleted from the corresponding tables on the admin site:
![image](https://github.com/kartoza/sawps/assets/90245715/eadb2056-4a23-4c1f-9322-b659494c20df)

If users are added to the organisation using the admin site, they do not reflect correctly on the front end organisation page (either they do not show or remain with the pending status even after joining)

**Comment by dimasciput** [comment]
> If users or managers are deleted here, they are not deleted from the corresponding tables on the admin site:

This should only remove the user's membership from this organization, not delete the user entirely. Do they want to delete the user as well? Because users can join multiple organizations.

**Comment by LunaAsefaw** [comment]
@dimasciput yes it is just to remove them from the organisation and not delete them.
But I have found an issue with this for managers, see #1976
